### PR TITLE
Adding Reviewers for Azure Documentation

### PR DESCRIPTION
**Pull request by berndverst**
Adding myself as a reviewer so I can help review docs, especially for Azure.

### CODE REVIEW

**Comment by berndverst** [comment]
Blocked by https://github.com/kubeflow/internal-acls/pull/239

File: content/docs/azure/OWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Azure Docs Owners File
+approvers:  # No new approvers
+reviewers:
+  - berndverst
```
Comment (sarahmaddox) [inline content/docs/azure/OWNERS:4]: @berndverst I think we should add you as a reviewer in the top-level OWNERS file for the kubeflow/website repository. Would you be happy with that solution? My thinking is that you're adding yourself as a reviewer, not an approver, which means that you're not acquiring the extra permissions that the approver role brings to the doc set as a whole. WDYT?

Comment (berndverst) [reply content/docs/azure/OWNERS:4]: Works for me. Let me do that instead!

**Comment by berndverst** [comment]
@sarahmaddox updated the PR! Thanks!

**Comment by sarahmaddox** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**